app/javascript/components/common/Introducer.tsx

```diff
@@ -30,17 +31,13 @@ export const Introducer = ({
     },
     {
       onSuccess: () => {
-        setHidden(true)
+        ref.current!.parentElement!.classList.add('hidden')
       },
     }
   )
 
-  if (hidden) {
-    return null
-  }
```

Comment:
We don't actually want the content to stop showing as we fade out the parent container instead, and we also need to persist the `ref` to access the container, so we continue to render the div.